src/components/picker/types.tsx

```diff
@@ -141,7 +141,11 @@ export type PickerBaseProps = Omit<NewTextFieldProps, 'value' | 'onChange'> & {
   /**
    * callback for picker modal search input text change (only when passing showSearch)
    */
-  onSearchChange?: (searchValue: string, filteredChildren?: ReactNode | undefined) => void;
+  onSearchChange?: (
+    searchValue: string,
+    filteredChildren?: ReactNode | undefined,
+    filteredItems?: Pick<PickerItemProps, 'label' | 'value' | 'disabled'>[]
```

Comment:
The user should not have to do something like:
```
onSearchChange={(value, _bla, changed) => {
  console.warn('value', value);
  console.warn('changed', changed);
}}
```
in order to get the changes.

You can simply change the name and type of the second value (this should be changed in the API of course)